ptero_workflow/implementation/models/methods/shell_command.py

```diff
@@ -160,8 +158,8 @@ def _shell_command_submit_url(self):
     def _shell_command_submit_data(self, colors, begins, execution_id):
         submit_data = self.parameters
         submit_data.update({
-            'stdin': json.dumps(
-                self.task.get_inputs(colors, begins)),
+           'commandLine': self._wrap_command_line(
+               submit_data['commandLine'], execution_id),
```

Comment:
I guess this also makes it harder to make the method code general for all registered services.